### PR TITLE
Changed SDK 0.8.0 to 1.0.0

### PR DESCRIPTION
**Pull request by rousso**

### CODE REVIEW

File: src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java [hunk 1]
```diff
@@ -42,7 +42,10 @@
 import eu.europa.ted.efx.model.Expression.StringListExpression;
 import eu.europa.ted.efx.model.Expression.TimeExpression;
 import eu.europa.ted.efx.model.Expression.TimeListExpression;
-import eu.europa.ted.efx.sdk0.v8.EfxParser.*;
+import eu.europa.ted.efx.sdk1.EfxBaseListener;
+import eu.europa.ted.efx.sdk1.EfxLexer;
+import eu.europa.ted.efx.sdk1.EfxParser;
```
Comment (bertrand-lorentz) [inline src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java:47]: Those 3 import statements are not needed: they refer to classes in the same package as the current class.

File: src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java [hunk 2]
```diff
@@ -59,7 +62,7 @@
  * {@link EfxExpressionTranslator08} only focuses on EFX expressions. To translate EFX templates
  * you need to use the {@link EfxTemplateTranslator08} which derives from this class.
  */
-@VersionDependentComponent(versions = {"0.8"}, componentType = VersionDependentComponentType.EFX_EXPRESSION_TRANSLATOR)
+@VersionDependentComponent(versions = {"1.0"}, componentType = VersionDependentComponentType.EFX_EXPRESSION_TRANSLATOR)
 public class EfxExpressionTranslator08 extends EfxBaseListener
```
Comment (bertrand-lorentz) [inline src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java:66]: This class should not be named EfxExpressionTranslator**08** anymore.
Same for the other classes *08 in this package

**Review by bertrand-lorentz — CHANGES_REQUESTED** [review]

File: src/main/java/eu/europa/ted/efx/sdk1/entity/SdkCodelist08.java [hunk 3]
```diff
@@ -10,7 +10,7 @@
  *
  * @author rouschr
  */
-@VersionDependentComponent(versions = {"0.8"}, componentType = VersionDependentComponentType.CODELIST)
+@VersionDependentComponent(versions = {"1.0"}, componentType = VersionDependentComponentType.CODELIST)
 public class SdkCodelist08 extends SdkCodelist {
```
Comment (rouschr) [inline src/main/java/eu/europa/ted/efx/sdk1/entity/SdkCodelist08.java:14]: SdkCodelist08, probably not 08 ?!

**Review by rouschr — CHANGES_REQUESTED** [review]
Same comment as Bertrand.

File: src/main/java/eu/europa/ted/efx/sdk1/entity/SdkCodelist08.java (hunk 3 above)
Comment (rousso) [reply src/main/java/eu/europa/ted/efx/sdk1/entity/SdkCodelist08.java]: Fixed

File: src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java (hunk 2 above)
Comment (rousso) [reply src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java]: Fixed

File: src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java (hunk 1 above)
Comment (rousso) [reply src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java]: Fixed

**Review by bertrand-lorentz — APPROVED** [review]

**Review by meletev — APPROVED** [review]

**Pull request merged**